.github/ionic-issue-bot.yml

```diff
@@ -48,6 +48,24 @@ stale:
     template is fully filled out.
 
 
+    Thank you for using Ionic!
+  close: true
+  lock: true
+  dryRun: false
+
+noReply:
+  days: 30
+  maxIssuesPerRun: 100
+  label: needs reply
+  responseLabel: triage
```

Comment:
experimented with using `triage` for the replies in my test repo, and I liked the workflow a lot better, especially since subsequent relabeling automagically removes the `triage` label.

Reply:
plus it puts the issue back through an existing workflow... (once we get fully into that workflow, anyhow)

Reply:
I like it